docs/advanced.rst

```diff
@@ -302,8 +306,8 @@ in a few ways:
    or 0.0.0.0.
 -  Set the ``ignore_hosts`` configuration option to a list of hosts to
    ignore
--  Add a ``before_record`` callback that returns None for requests you
```

Comment:
should we deprecate `before_record`? https://github.com/kevin1024/vcrpy/blob/2156adb841241e724667b2c9d654f1ed2be22752/vcr/config.py#L203